optimism.tokenlist.json

```diff
@@ -6,7 +6,7 @@
     "layer2",
     "infrastructure"
   ],
-  "timestamp": "2022-08-18T21:20:01.549Z",
+  "timestamp": "2022-08-18T22:12:28.616Z",
```

Comment:
Hi, please do not update the token list directly. We update the token list automatically when PRs are merged into master. Otherwise this LGTM!

Reply:
Hi, thanks for the prompt reply. No problem, these actions are run automatically since they're part of the repo. So, they're run in the fork as well, but I can disable actions manually and remove the `bot(ci)` commits. Does that work?

Reply:
Ohhh, I see the issue. One sec.

Reply:
https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/143 should resolve this, thanks for letting us know!

Reply:
Thank you! Just cleaned up and removed the tokenlist commits, it should only be the two files now.